samples/bot-formatting-cards/nodejs/Readme.md

```diff
@@ -113,6 +114,47 @@ Run ngrok - point to port 3978
             ]
         ```
 
+  **Note: In Adaptive Card what we are defining (User details) should be exist in the same tenant where you are testing the app (teams login) etc...**
```

Comment:
**Note: In adaptive card, what we are defining (User details) should be exist in the same tenant where you are testing the app (teams' login) etc...**